data/markdown/pages/learn/faq/xm-cloud/index.md

```diff
@@ -27,7 +27,7 @@ Sitecore XM Cloud is a cloud-native, hybrid headless CMS, for developers and mar
 - Pages content editor
 - Sitecore Experience Accelerator (SXA)
 - Headless Services
-- Sitecore Next.js SDK
+- Sitecore SDKs including Next.JSS, .NET Core and Angular
```

Comment:
These names have typos, and have incorrect casing.

Can we change this to: `Next.js, ASP.NET Core and Angular`